tracking-with-geofence-notifications/app/src/main/java/software/amazon/location/sample/viewModel/MainViewModel.kt

```diff
@@ -38,7 +31,6 @@ import software.amazon.location.tracking.util.BackgroundTrackingMode
 import software.amazon.location.tracking.util.Logger
 import software.amazon.location.tracking.util.ServiceCallback
 import software.amazon.location.tracking.util.StoreKey
-import java.util.Date
 
 class MainViewModel : ViewModel() {
     var locationTracker: LocationTracker? = null
```

Comment:
Is there a way to handle this case without setting any values as null?

Reply:
Code optimized. For null, the above approach is widely used, and also I am using the safe (?.) operator while accessing to avoid crashes.